internal/rbd/rbd_util.go

```diff
@@ -212,6 +244,14 @@ type migrationVolID struct {
 	clusterID string
 }
 
+type rbdQos struct {
```

Comment:
remove the rbd prefixes here too please, this is all in the rbd package no need for the duplication

Reply:
done